mmv1/products/compute/api.yaml

```
@@ -4114,6 +4114,17 @@ objects:
           The internal fully qualified service name for this Forwarding Rule.
           This field is only used for INTERNAL load balancing.
         output: true
+      - !ruby/object:Api::Type::String
+        name: 'pscConnectionId'
+        description: |
+          [Output Only] The PSC connection id of the PSC Forwarding Rule in uint64 format.
```

```suggestion
          The PSC connection id of the PSC Forwarding Rule in uint64 format.
```

Remove [Output Only], documentation automation handles this